test/gtest/ucp/test_ucp_worker.cc

```diff
@@ -462,3 +462,88 @@ UCS_TEST_P(test_ucp_worker_discard, wireup_ep_flush_ok_not_wait_comp) {
 }
 
 UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_worker_discard, all, "all")
+
+
+class test_ucp_worker_request_leak : public ucp_test {
+public:
+    enum {
+        LEAK_CHECK,
+        LEAK_IGNORE
+    };
+
+    static void get_test_variants(std::vector<ucp_test_variant> &variants)
+    {
+        add_variant_with_value(variants, UCP_FEATURE_TAG, LEAK_CHECK,
+                               "leak_check");
+        add_variant_with_value(variants, UCP_FEATURE_TAG, LEAK_IGNORE,
+                               "leak_ignore");
+    }
+
+    bool ignore_leak()
+    {
+        return get_variant_value(0) == LEAK_IGNORE;
+    }
+
+    /// @override
```

Comment:
minor extra `/` here and below

Reply:
it's supposed to be doxgen comment
https://www.doxygen.nl/manual/docblocks.html

> A third alternative is to use a block of at least two C++ comment lines, where each line starts with an additional slash or an exclamation mark. Here are examples of the two cases:
> 
> ///
> /// ... text ...
> ///

Reply:
aa, ok